net/ioutil.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import { BufReader } from "./bufio.ts";
+
+export async function readShort(buf: BufReader) {
+  const [high, low] = [await buf.readByte(), await buf.readByte()];
+  return (high << 8) | low;
+}
+
+export async function readInt(buf: BufReader) {
+  const [high, low] = [await readShort(buf), await readShort(buf)];
+  return (high << 16) | low;
+}
```

Comment:
I don't want to trouble you too much for this initial code dump, but it would be very nice to have some basic tests for these functions in `net/ioutil_test.ts` ... If it's getting too much feel free to ignore this and we'll do it later.

Reply:
Very thank you to mention about it! `readLong` and `sliceLongToBytes` were buggy because of  problem around ECMAScript's 64 bit bit operations. I fixed and wrote tests.